script.js

```diff
@@ -16,3 +16,201 @@ document.querySelectorAll('.nav_link').forEach((n) => n.addEventListener('click'
   navBranding.classList.remove('active');
   navContainer.classList.remove('active');
 }));
+
+// multistories project
+
+const multiPost = {
```

Comment:
It seems you forget the mobile version of your popup. It seems totally broken, there is an horizontal overflow. Kindly fix it, it needs to be like on the Figma template. Check the GIF below to understand the issue better 👍 
<img width="385" alt="Screenshot 2023-04-13 at 9 04 21 PM" src="https://user-images.githubusercontent.com/25887253/231848366-46b00d4e-824c-454d-a894-2e0e7a08f924.png">

![Screen Recording 2023-04-13 at 8 56 33 PM](https://user-images.githubusercontent.com/25887253/231848346-972dd2c6-4af4-4d56-b1a0-b078c74a696e.gif)